packages/react-components/react-breadcrumb/src/components/BreadcrumbDivider/useBreadcrumbDivider.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { getNativeElementProps } from '@fluentui/react-utilities';
+import type { BreadcrumbDividerProps, BreadcrumbDividerState } from './BreadcrumbDivider.types';
+import { ChevronRight20Regular, ChevronRight16Regular, ChevronRight12Regular } from '@fluentui/react-icons';
+import { BreadcrumbProps } from '../Breadcrumb/Breadcrumb.types';
+import { useBreadcrumbContext_unstable } from '../Breadcrumb/BreadcrumbContext';
+
+/**
+ * Create the state required to render BreadcrumbDivider.
+ *
+ * The returned state can be modified with hooks such as useBreadcrumbDividerStyles_unstable,
+ * before being passed to renderBreadcrumbDivider_unstable.
+ *
+ * @param props - props from this instance of BreadcrumbDivider
+ * @param ref - reference to root HTMLElement of BreadcrumbDivider
+ */
+export const useBreadcrumbDivider_unstable = (
+  props: BreadcrumbDividerProps,
+  ref: React.Ref<HTMLElement>,
```

Comment:
```suggestion
  ref: React.Ref<HTMLLIElement>,
```

Should it be `HTMLLIElement`?